src/lib.rs

```diff
@@ -34,8 +34,13 @@
 //! render all CSRF mitigation techniques ineffective!
 //!
 //! ## Features:
-//! - `actix-multipart` Enables compatibility with
-//!     the [`actix_multipart`] library.
+#![cfg_attr(
+    feature = "actix-multipart",
+    doc = r##"
+- `actix-multipart` Enables compatibility with
+    the [`actix_multipart`] library.
+"##
+)]
```

Comment:
Feature visibility wise, wouldn't it be better to keep this ungated, so people without the feature can find it and add it in? Otherwise it's a little more difficult to find the feature.

I'm ok either way -- just wanted to hear your thoughts on that

Reply:
I'm not really experienced with this, so I'll leave it to your discretion on what you want to do there then.

Reply:
I decided to remove the feature-gate and the link, since that makes the list more easy to use offline.